lib/site_template/_config.yml

```diff
@@ -29,6 +29,15 @@ markdown: kramdown
 theme: minima
 gems:
   - jekyll-feed
-exclude:
-  - Gemfile
-  - Gemfile.lock
+
+# Exclude from processing.
+# The following items will not be processed, by default. Create a custom list
+# to override the default setting.
+# exclude:
```

Comment:
@pathawks, it just struck me.. we could have the list automatically updated if `_config.yml` was generated from an ERB template `_config.yml.erb` : 

```
# exclude:
<% for item in Jekyll::Configuration::DEFAULTS["exclude"] %>
#   - <%= item %>
<% end %>
```
concern: Matt's [comment](https://github.com/jekyll/jekyll/pull/5236#issuecomment-243956535) from many months ago regarding the use of ERB templates..

Your thoughts ?